src/documentation/gtfs_to_ntfs_specs.md

```diff
@@ -178,6 +179,17 @@ specified, the conversion should stop immediately with an error.
 
 (1) When several GTFS Routes with different `route_type`s are grouped together, the commercial_mode_id with the smallest priority should be used (as specified in chapter "Mapping of route_type with modes").
 
+### Reading calendars.txt and calendar_dates.txt
+Dates of service are trasnformed into explicit active service exceptions as if using a single NTFS file calendar_dates.txt. The resulting files might be different following an optimization operation applied at the end of the conversion, but the result should be functionally identical.
+* In case both files calendar.txt and calendar_dates.txt are present in the input dataset, the days of the week of the specified services within the date range [`start_date` - `end_date`] are transformed into explicit active service dates, taking into account the dates when service exceptions occur. Note that the generated (`service_id`, `date`) pairs must be unique.
```

Comment:
Similar to what @datanel already mentioned above.
```suggestion
* In case both files `calendar.txt` and `calendar_dates.txt` are present in the input dataset, the days of the week of the specified services within the date range [`start_date` - `end_date`] are transformed into explicit active service dates, taking into account the dates when service exceptions occur. Note that the generated (`service_id`, `date`) pairs must be unique.
```